website/static/_redirects

```diff
@@ -742,6 +743,9 @@ https://tutorial.getdbt.com/* https://docs.getdbt.com/:splat 301!
 /v0.13/docs/requiring-dbt-versions / 301
 /v0.14/docs/cloud-developer-ide / 301
 
+/v0.13/docs/building-a-new-adapter /guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter 301
+/v0.15/docs/configure-your-profile /docs/core/connection-profiles 301
```

Comment:
I'm seeing the same issue here. @runleonarun can you also confirm the destination here is the correct one we want folks to land on?

Reply:
I deleted the **/docs/building-a-new-adapter /docs/contributing/building-a-new-adapter 301** since the new doc wants this to point to **/docs/building-a-new-adapter /guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter 301**

I also moved /v0.13/docs/building-a-new-adapter /guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter 301 above /docs/building-a-new-adapter /guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter 301 

I am guessing this is the correct order?:

Specific
/v0.13/docs/building-a-new-adapter /guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter 301

General:
/docs/building-a-new-adapter /guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter 301